state/protocol/badger/mutator.go

```diff
@@ -694,50 +685,10 @@ func (m *FollowerState) epochStatus(block *flow.Header) (*flow.EpochStatus, erro
 	if parentSetup.FinalView < block.View { // first block of a new epoch
 		// sanity check: parent's epoch Preparation should be completed and have EpochSetup and EpochCommit events
 		if parentStatus.NextEpoch.SetupID == flow.ZeroID {
-			//return nil, fmt.Errorf("missing setup event for starting next epoch")
-
-			// TMP: EMERGENCY EPOCH CHAIN CONTINUATION
-			//
-			// We are proposing or processing the first block of the next epoch,
-			// but that epoch has not been setup. Rather than returning an error
-			// which prevents further block production, we store the block with
-			// the same epoch status as its parent, resulting in it being considered
-			// by the protocol state to fall in the same epoch as its parent.
-			//
-			// CAUTION: this is inconsistent with the FinalView value specified in the epoch.
-			//
-			status, err := flow.NewEpochStatus(
-				parentStatus.PreviousEpoch.SetupID, parentStatus.PreviousEpoch.CommitID,
-				parentStatus.CurrentEpoch.SetupID, parentStatus.CurrentEpoch.CommitID,
-				parentStatus.NextEpoch.SetupID, parentStatus.NextEpoch.CommitID,
-			)
-			if err != nil {
-				return nil, fmt.Errorf("failed to create epoch status for fallback epoch: %w", err)
-			}
-			return status, fmt.Errorf("entering not set up epoch: %w", errEmergencyEpochChainContinuation)
+			return nil, fmt.Errorf("missing setup event for starting next epoch: %w", errIncompleteEpochConfiguration)
```

Comment:
consolidated all EECC logic in one place in the `handleServiceEvents` method:  https://github.com/onflow/flow-go/blob/76127e13b9a05ba62c84b7f3b349f5f131998a04/state/protocol/badger/mutator.go#L750-L766